reporters_db/data/reporters.json

```diff
@@ -1401,15 +1401,22 @@
             "editions": {
                 "Bankr. L. Rep.": {
                     "end": null,
+                    "regexes": [
+                        "$full_cite_paragraph"
+                    ],
                     "start": "1750-01-01T00:00:00"
                 }
             },
+            "examples": [
+                "Bankr. L. Rep. P12,345",
+                "Bankr. L. Rep. P 80679",
+                "Bankr. L. Rep. (CCH) ¶ 67,488",
+                "Bankr. L. Rep. (C.C.H.) ¶67,049"
+            ],
```

Comment:
I mean, this is cool. One line to add a regex that matches all those weird examples. Cool.